lib/src/main/java/org/team100/lib/motor/turning/Falcon6TurningMotor.java

```diff
@@ -34,18 +34,18 @@ public class Falcon6TurningMotor implements Motor100<Angle100> {
      * The speed, below which, static friction applies, in motor revolutions per
      * second.
      */
-    private static final double staticFrictionSpeedLimitRev_S = 1.0;
+    private static final double staticFrictionSpeedLimitRev_S = 0.1;
 
     /**
      * Friction feedforward in volts, for when the mechanism is stopped, or nearly
      * so.
      */
-    private static final double staticFrictionFFVolts = 0.375;
+    private static final double staticFrictionFFVolts = 0.375/20;
```

Comment:
don't do this sort of math in constants, just write what you think the value should be.